mcp/streamable.go

```diff
@@ -166,7 +167,7 @@ func (h *StreamableHTTPHandler) ServeHTTP(w http.ResponseWriter, req *http.Reque
 			h.mu.Lock()
 			delete(h.transports, transport.SessionID)
 			h.mu.Unlock()
-			transport.connection.Close()
+			_ = transport.session.Close()
```

Comment:
Please add `// TODO: consider logging this error` here, to make it easier to find when we audit observability.

Reply:
Yeah good point. I just added the TODO via eab40f7